lib/EasyPost/Webhook.php

```diff
@@ -82,4 +84,37 @@ public static function create($params = null, $apiKey = null)
 
         return self::createResource(get_class(), $params, $apiKey);
     }
+
+    /**
+     * Validate a webhook originated from EasyPost by comparing the HMAC header to a shared secret.
+     * If the signatures do not match, an error will be raised signifying the webhook either did not originate
+     * from EasyPost or the secrets do not match. If the signatures do match, the `event_body` will be returned as JSON.
+     *
+     * @param mixed $eventBody
+     * @param mixed $headers
+     * @param string $webhookSecret
+     * @return mixed
+     */
+    public static function validateWebhook($eventBody, $headers, $webhookSecret)
+    {
+        $easypostHmacSignature = $headers["X-Hmac-Signature"] ?? null;
+
+        if ($easypostHmacSignature != null) {
+            $normalizedSecret = Normalizer::normalize($webhookSecret, Normalizer::FORM_KD);
+            $encodedSecret = mb_convert_encoding(utf8_encode($normalizedSecret), "ISO-8859-1", "UTF-8");
```

Comment:
This still isn't perfectly ideal as there could be slight alterations depending on the input. Let's fix it to be uniform with the other libraries.
```suggestion
            $encodedSecret = mb_convert_encoding($normalizedSecret, 'UTF-8');
```

Reply:
fixed